[2023-03-30 18:15:13,570] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: elastic_desk.popula_elastic_aguardando scheduled__2023-03-30T18:10:00+00:00 [queued]>
[2023-03-30 18:15:13,584] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: elastic_desk.popula_elastic_aguardando scheduled__2023-03-30T18:10:00+00:00 [queued]>
[2023-03-30 18:15:13,584] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-03-30 18:15:13,584] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-03-30 18:15:13,584] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-03-30 18:15:13,604] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): popula_elastic_aguardando> on 2023-03-30 18:10:00+00:00
[2023-03-30 18:15:13,610] {standard_task_runner.py:52} INFO - Started process 1759 to run task
[2023-03-30 18:15:13,616] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'elastic_desk', 'popula_elastic_aguardando', 'scheduled__2023-03-30T18:10:00+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/etl_elastic_desk.py', '--cfg-path', '/tmp/tmpghzbrms3', '--error-file', '/tmp/tmpkrwyzndv']
[2023-03-30 18:15:13,617] {standard_task_runner.py:80} INFO - Job 41: Subtask popula_elastic_aguardando
[2023-03-30 18:15:13,715] {task_command.py:371} INFO - Running <TaskInstance: elastic_desk.popula_elastic_aguardando scheduled__2023-03-30T18:10:00+00:00 [running]> on host 20743fec282a
[2023-03-30 18:15:13,846] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clecio.antao@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=elastic_desk
AIRFLOW_CTX_TASK_ID=popula_elastic_aguardando
AIRFLOW_CTX_EXECUTION_DATE=2023-03-30T18:10:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-30T18:10:00+00:00
[2023-03-30 18:15:13,884] {base.py:271} INFO - GET http://192.168.0.100:9200/ [status:200 request:0.015s]
[2023-03-30 18:15:16,238] {base.py:271} INFO - HEAD http://192.168.0.100:9200/_template/desk_aguardando [status:200 request:0.005s]
[2023-03-30 18:15:16,517] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/es_pandas/es_pandas.py:58 RuntimeWarning: invalid value encountered in double_scalars
[2023-03-30 18:15:16,891] {base.py:271} INFO - POST http://192.168.0.100:9200/_bulk [status:200 request:0.342s]
[2023-03-30 18:15:16,924] {python.py:173} INFO - Done. Returned value was: None
[2023-03-30 18:15:16,943] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=elastic_desk, task_id=popula_elastic_aguardando, execution_date=20230330T181000, start_date=20230330T181513, end_date=20230330T181516
[2023-03-30 18:15:16,999] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-30 18:15:17,058] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
